projects/energondex/index.js

```diff
@@ -0,0 +1,23 @@
+const ADDRESSES = require('../helper/coreAssets.json')
+const { getUniTVL, staking } = require('../helper/unknownTokens')
+const sdk = require('@defillama/sdk')
+const uniTvl = getUniTVL({ factory: '0xBfAe4f07C099798F23f5aC6773532fB637B68Ad7', useDefaultCoreAssets: true,  })
+const masterchef  = '0x8b804321b8D094D8C9bB3bFF8CC580087E8d13E0'

```

Comment:
this address is wrong? https://basescan.org/address/0x8b804321b8D094D8C9bB3bFF8CC580087E8d13E0

Reply:
https://basescan.org/address/0x9a40f8374cf840cb1d76dd085e250d86641fb6a3 This is the correct one